crates/esthri-cli/Cargo.toml

```diff
@@ -56,4 +56,4 @@ backtrace = "0.3"
 fs_extra = "1.2"
 md5 = "0.7"
 tempdir = "0.3"
-zip = "0.6.2"
+zip = "0.5"
```

Comment:
Version conflict forced me to downgrade this library:

```
error: failed to select a version for `zstd-sys`.
    ... required by package `zstd-safe v4.1.4+zstd.1.5.2`
    ... which satisfies dependency `zstd-safe = "=4.1.4"` of package `zstd v0.10.0+zstd.1.5.2`
    ... which satisfies dependency `zstd = "^0.10.0"` of package `zip v0.6.0`
    ... which satisfies dependency `zip = "^0.6"` of package `esthri-cli v8.4.0 (/home/ubuntu/dev/esthri/crates/esthri-cli)`
versions that meet the requirements `=1.6.3` are: 1.6.3+zstd.1.5.2

the package `zstd-sys` links to the native library `zstd`, but it conflicts with a previous package which links to `zstd` as well:
package `zstd-sys v2.0.1+zstd.1.5.2`
    ... which satisfies dependency `zstd-sys = "=2.0.1"` of package `zstd-safe v5.0.1+zstd.1.5.2`
    ... which satisfies dependency `zstd-safe = "^5.0.1"` of package `async-compression v0.3.14`
    ... which satisfies dependency `async-compression = "^0.3.14"` of package `async_zip v0.0.8`
    ... which satisfies dependency `async_zip = "^0.0.8"` of package `esthri-cli v8.4.0 (/home/ubuntu/dev/esthri/crates/esthri-cli)`
Only one package in the dependency graph may specify the same links value. This helps ensure that only one copy of a native library is linked in the final binary. Try to adjust your dependencies so that only one package uses the links ='zstd-sys' value. For more information, see https://doc.rust-lang.org/cargo/reference/resolver.html#links.

failed to select a version for `zstd-sys` which could resolve this conflict
[Finished running. Exit status: 101]
```